bin/systemFonts.js

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const DEFAULT_WEIGHTS = [400, 700];
+const DEFAULT_STYLES = ['italic', 'regular'];
+
+const SYSTEM_FONTS = [
```

Comment:
This could be shortened a bit by sth like:
```
const DEFAULT_SYSTEM_ATTS = {
  service: 'system',
  weights: DEFAULT_WEIGHTS,
  styles: DEFAULT_STYLES,
};
```
and then just using
`...DEFAULT_SYSTEM_ATTS` for all.

Reply:
Makes sense, less error-prone this way 👍